src/misc/programs.cpp

```diff
@@ -376,6 +376,89 @@ class CONFIG final : public Program {
 	}
 };
 
+static char msg[200];
+const char *SetProp(std::vector<std::string> &pvars) {
+	*msg = 0;
+	// attempt to split off the first word
+	std::string::size_type spcpos = pvars[0].find_first_of(' ');
+	std::string::size_type equpos = pvars[0].find_first_of('=');
+
+	if ((equpos != std::string::npos) &&
+		((spcpos == std::string::npos) || (equpos < spcpos))) {
+		// If we have a '=' possibly before a ' ' split on the =
+		pvars.insert(pvars.begin()+1,pvars[0].substr(equpos+1));
+		pvars[0].erase(equpos);
+		// As we had a = the first thing must be a property now
+		Section* sec=control->GetSectionFromProperty(pvars[0].c_str());
+		if (sec) pvars.insert(pvars.begin(),std::string(sec->GetName()));
+		else {
+			strcpy(msg, MSG_Get("PROGRAM_CONFIG_PROPERTY_ERROR"));
```

Comment:
Best to use `safe_strcpy` here. This will blow up if a particularly lengthy translation is required, and translators shouldn't have to know (or care) buffer sizes.

Reply:
Done.